doc/source/operations/upgrading-ceph.rst

```diff
@@ -159,6 +159,18 @@ versions``. Once confirmed, run the following command:
 
    ceph osd require-osd-release squid
 
+.. warning::
```

Comment:
Should/Can we do this before the upgrade? if so, could we move the warning to the top of the section?